### PR TITLE
Update changelog with fixes and improvements since v0.0.6

### PR DESCRIPTION
**Pull request by rwjuk**
Hopefully I didn't miss any!

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -5,14 +5,53 @@
...
+- Improved: Remove duplicate ride penalty for closed rides.
+- Improved: Make shortcut keys window larger and resizable.
+- Removed: known_issues.txt no longer used, check issue tracker on GitHub.
+- Fix: [#1992] Felicity Anderson Cheat can crash the game, as well as blocking queues.
+- Fix: [#4493] Provide tooltip for disabled price field.
 - Fix: [#4689] Object selection tabs sometimes flicker.
 - Fix: [#4913] Server greeting displaying local setting.
+- Fix: [#4972] Map window not updated properly when shrinking map from Map Generation window.
+- Fix: [#5004] Peeps in parks imported from RCT1 show abnormalities.
+- Fix: [#5014] Research not importect from RCT1 correctly.
```
Comment (duncanspumpkin) [inline distribution/changelog.txt]: Spelling

File: distribution/changelog.txt [hunk 2]
```diff
@@ -5,14 +5,53 @@
...
+- Fix: [#5199] "Force a breakdown" debugging tool isn't hidden in multiplayer.
 - Fix: [#5188] Clicking on a Magic Carpet doesn't open the ride window.
+- Fix: [#5218] Scale RCT1 park value objectives.
+- Fix: [#5219] Game crashes when opening 'misc' tab in options.
 - Fix: [#5238] RCT1 import: Rides are initially free when placing them.
-- Fix: [#5299] Scenario editor crash when placing peep spawn
+- Fix: [#5252] Correct typo in Conger Eel Coaster description.
+- Fix: [#5265] Queue line TVs not detected properly.
+- Fix: [#5271] Keyboard shortcuts window isn't large enough (for some languages).
+- Fix: [#5275] Door animations do not play correctly.
```
Comment (duncanspumpkin) [inline distribution/changelog.txt]: This fixed A bug that i introduced as part of this release so it doesn't need to be mentioned

File: distribution/changelog.txt [hunk 3]
```diff
@@ -5,14 +5,53 @@
...
+- Fix: [#5284] Mechanic is called to fix a ride that's outside his patrol area.
+- Fix: [#5285] Intro always plays even if play_intro = false.
+- Fix: [#5299] Scenario editor crash when placing peep spawn.
+- Fix: [#5316] Banners always spawn black.
```
Comment (duncanspumpkin) [inline distribution/changelog.txt]: Same for this

File: distribution/changelog.txt (hunk 1 above)
Comment (rwjuk) [inline distribution/changelog.txt]: That'll teach me to copy-paste from the issue tracker!

**Comment by rwjuk** [comment]
@duncanspumpkin As @IntelOrca already merged it, should I create a new PR for the fixes or do you just want to make them yourself?

**Comment by IntelOrca** [comment]
I'll do it

**Pull request merged**